### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by BorntraegerMarc**

### CODE REVIEW

**Comment by BorntraegerMarc** [comment]
@wzrdtales I would like to suggest to remove the commitlint bot. I have literally no clue what I'm doing wrong. I added the text "Signed-off-by: Marc Bornträger <marc.borntraeger@gmail.com>" to the commit message but it does not accept it. It's a bit annoying and complicates the contribution process 😄

**Comment by wzrdtales** [comment]
@BorntraegerMarc These are two problems, one is the DCO the other is that you did not followed conventional commits. https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-angular/convention.md

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Minimal reproduction of the problem with instructions
+
+## What is the motivation / use case for changing the behavior?
+<!-- Describe the motivation or the concrete use case. -->
+
+
+## Environment
+
+<pre><code>
+db-migrate version: X.Y.Z
```
Comment (wzrdtales) [inline .github/ISSUE_TEMPLATE.md]: used plugins, and driver versions should be noted too.

**Comment by BorntraegerMarc** [comment]
@wzrdtales the URL you've posted resolves into a 404

**Pull request closed without merging**